imagersite/imager_images/templates/imager_images/album_edit.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{% load images_filters %}
```

Comment:
you're the first student I've had to figure out an utilize custom filters.  Nice!